webapp/app/components/pages/login/LoginHOC.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
```

Comment:
Could not find the test for this component, does it have one?

Reply:
Done!